pkg/kubelet/kubelet.go

```diff
@@ -1386,7 +1386,11 @@ func (kl *Kubelet) GetKubeletContainerLogs(podFullName, containerName, tail stri
 		// waiting state.
 		return err
 	}
-	return kl.containerManager.GetContainerLogs(containerID, tail, follow, stdout, stderr)
+	pod, ok := kl.GetPodByFullName(podFullName)
```

Comment:
I'd be fine with adding a TODO to re-evaluate and remove the redundancy later, if it's preferred :)